### DISCUSSION TITLE
v2.8.2

### DISCUSSION THREAD

**Discussion by MAKOMO** [artisan-roaster-scope/artisan#1055] [Versions]
This version adds support for [Sivetz fluid bed roasting machines](https://artisan-scope.org/machines/sivetz/), [Santoker Q Series and R Series roasters](https://artisan-scope.org/machines/santoker/), the [Yocto Watt module](https://artisan-scope.org/devices/yoctopuce/#Yocto-Watt), the [Phidget DAQ1500](https://artisan-scope.org/devices/phidgets/#DAQ1500), and speeds up the [Designer](https://artisan-roasterscope.blogspot.com/2019/05/using-artisan-designer.html).

The [Artisan project](https://artisan-scope.org/) runs on donations from individuals and companies recognising the value of Artisan for their work or leisure. Consider supporting this project with your [donation](https://www.paypal.com/paypalme/MarkoLuther), or even better, purchase an [artisan.plus membership](https://shop.artisan.plus/)!

## Documentation

__>>>  [Release Notes](https://artisan-roasterscope.blogspot.com/2022/12/artisan-v282.html) <<<__

See the [detailed release history](https://github.com/artisan-roaster-scope/artisan/blob/master/wiki/ReleaseHistory.md) for a complete list of additions, changes and fixes and the [Installation Instructions](https://github.com/artisan-roaster-scope/artisan/blob/master/wiki/Installation.md) for platform specific notes. Read our [Quick-Start Guide](https://artisan-scope.org/docs/quick-start-guide/)!

## Supported Platforms

Builds are available for macOS Catalina 10.15, Windows 10 (x64), Linux glibc 2.31 and 32bit Raspbian Bullseye or newer.

Legacy builds for macOS High Sierra 10.13 and Windows 8 (x64) are available too.

---

PLEASE FOLLOW AND TAG US!  
&nbsp;&nbsp;&nbsp;&nbsp;<a href="https://www.facebook.com/ArtisanScope"><img src="https://github.com/artisan-roaster-scope/artisan/blob/master/wiki/screenshots/facebook-square.svg?raw=true" width="30"></a>&nbsp;&nbsp;&nbsp;<a href="https://www.instagram.com/artisanscope/"><img src="https://github.com/artisan-roaster-scope/artisan/blob/master/wiki/screenshots/instagram-square.svg?raw=true" width="30"></a>&nbsp;&nbsp;&nbsp;<a href="https://www.youtube.com/channel/UCEp-DH_Y7sBR3-X3FWomgog"><img src="https://github.com/artisan-roaster-scope/artisan/blob/master/wiki/screenshots/youtube-svgrepo-com.svg?raw=true" width="30"></a>&nbsp;&nbsp;&nbsp;<a href="https://masto.ai/@artisan"><img src="https://github.com/artisan-roaster-scope/artisan/blob/master/wiki/screenshots/mastodon-icon-svgrepo-com.svg?raw=true" width="30"></a>&nbsp;&nbsp;&nbsp;<a href="https://www.linkedin.com/company/artisan-plus"><img src="https://github.com/artisan-roaster-scope/artisan/blob/master/wiki/screenshots/linkedin-svgrepo-com.svg?raw=true" width="30"></a>
---

__Licence Notes__

_Artisan is free software: you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation, either version 3 of the License, or
(at your option) any later version._

_Artisan is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE._

_Copies of the GNU General Public License has been included with this 
distribution in the file `LICENSE.txt`. An online version is available at
<http://www.gnu.org/licenses/>._

<hr /><em>This discussion was created from the release <a href='https://github.com/artisan-roaster-scope/artisan/releases/tag/v2.8.2'>v2.8.2</a>.</em>

**Comment by Terracotta-62** [comment]
No result was recorded during the modbus scan。
![9012b677f0b195c19c4f239133f0c81](https://user-images.githubusercontent.com/116416332/209151840-f9904062-7dfa-4c3b-9c5e-b4aa5aabbf80.png)

**Reply by MAKOMO** [reply]
Yep, MODBUS scan is broken in v2.8.2. I created a corresponding issue (Issue #1056 1056)

**Comment by arefkhodabandeh** [comment]
excuse me, how i can install this version on my mac?

**Reply by poundy** [reply]
this version is not a current release. You must use a current release, or you can build the app from source if you feel you must.... https://github.com/artisan-roaster-scope/artisan/releases/tag/v2.10.4 is the current release and has appropriate build availability.
